src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -98,7 +98,7 @@ func kubectl(stdin io.Reader, context *config.Context, args ...string) error {
 				return errors.Wrapf(err, "failed to discover default kube config: %q does not exist", kubeconfig)
 			}
 		}
-		kubeconfig = fmt.Sprintf("%s:%s", kubeconfig, tmpfile.Name())
+		kubeconfig = fmt.Sprintf("%s;%s", kubeconfig, tmpfile.Name())
```

Comment:
This'll break on unices. We need conditional logic to use `;` on windows, and `:` everywhere else

Reply:
`os.PathListSeparator` will contain the correct separator for the operating system you're compiling for.

Reply:
Riiight - fixed